anaconda.spec.in

```diff
@@ -189,6 +189,7 @@ for live installations.
 
 %package install-env-deps
 Summary: Installation environment specific dependencies
+BuildArchitectures: noarch
```

Comment:
I wonder, could this be a noarch package when it has arch specific conditions on requires list? Is it fine to do?

Reply:
Good point.
If the subpackage has architecture-specific conditions (like Requires for certain architectures), it should not be marked as noarch.

Reply:
I have removed `noarch` changes and added those packages to rpmlint whitelist.